methods_and_recursion/mad_libs.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+def ask_question(question)
+  puts question
+  gets.chomp
+end
+
+adjective_one = ask_question('Give me an adjective:')
+adjective_two = ask_question('Give me another adjective:')
+noun_one = ask_question('Give me a noun:')
+noun_two = ask_question('Give me another noun:')
+noun_three = ask_question('Give me a plural noun:')
+game = ask_question('Give me a game:')
+noun_four = ask_question('Give me another plural noun:')
+verb_one = ask_question("Give me a verb ending in 'ing':")
+verb_two = ask_question("Give me another verb ending in 'ing':")
+noun_five = ask_question('Give me another plural noun:')
+verb_three = ask_question("Give me another verb ending in 'ing':")
+noun_six = ask_question('Give me another noun:')
+plant = ask_question('Give me a plant:')
+body_part = ask_question('Give me a part of the body:')
+place = ask_question('Give me a place:')
+verb_four = ask_question("'Give me another verb ending in 'ing':")
+adjective_three = ask_question('Give me another adjective:')
+number = ask_question('Give me a number:')
+noun_seven = ask_question('Give me another plural noun:')
+
+puts "
```

Comment:
FYI... You can use a HEREDOC here.

    puts <<END_OF_MADLIB
        A vacation is when you take a trip...
        ...
    END_OF_MADLIB